tests/test_api.rs

```diff
@@ -3612,3 +3618,90 @@ fn date() {
   assert_eq!(date.value_of(), 3.0);
   assert_eq!(date.number_value(scope).unwrap(), 3.0);
 }
+
+#[test]
+fn symbol() {
+  let _setup_guard = setup();
+  let isolate = &mut v8::Isolate::new(Default::default());
+  let scope = &mut v8::HandleScope::new(isolate);
+  let context = v8::Context::new(scope);
+  let scope = &mut v8::ContextScope::new(scope, context);
+
+  let desc = v8::String::new(scope, "a description").unwrap();
+
+  let s = v8::Symbol::new(scope, None);
+  assert!(s.description(scope) == v8::undefined(scope));
+
+  let s = v8::Symbol::new(scope, Some(desc));
+  assert!(s.description(scope) == desc);
+
+  let s_pub = v8::Symbol::r#for(scope, Some(desc));
+  assert!(s_pub.description(scope) == desc);
+  assert!(s_pub != s);
+
+  let s_pub2 = v8::Symbol::r#for(scope, Some(desc));
+  assert!(s_pub2 != s);
+  assert!(s_pub == s_pub2);
+
+  let s = eval(scope, "Symbol.asyncIterator").unwrap();
+  assert!(s == v8::Symbol::get_async_iterator(scope));
+}
+
+#[test]
+fn private() {
+  let _setup_guard = setup();
+  let isolate = &mut v8::Isolate::new(Default::default());
+  let scope = &mut v8::HandleScope::new(isolate);
+  let context = v8::Context::new(scope);
+  let scope = &mut v8::ContextScope::new(scope, context);
+
+  let p = v8::Private::new(scope, None);
+  assert!(p.name(scope) == v8::undefined(scope));
+
+  let name = v8::String::new(scope, "a name").unwrap();
+  let p = v8::Private::new(scope, Some(name));
+  assert!(p.name(scope) == name);
+
+  let p_api = v8::Private::for_api(scope, Some(name));
+  assert!(p_api.name(scope) == name);
+  assert!(p_api != p);
+
+  let p_api2 = v8::Private::for_api(scope, Some(name));
+  assert!(p_api2 != p);
+  assert!(p_api == p_api2);
+}
+
+#[test]
+fn bigint() {
+  let _setup_guard = setup();
+  let isolate = &mut v8::Isolate::new(Default::default());
+  let scope = &mut v8::HandleScope::new(isolate);
+  let context = v8::Context::new(scope);
+  let scope = &mut v8::ContextScope::new(scope, context);
+
+  let b = v8::BigInt::new_from_u64(scope, 1337);
+  assert_eq!(b.u64_value(), (1337, true));
+
+  let b = v8::BigInt::new_from_i64(scope, -1337);
+  assert_eq!(b.i64_value(), (-1337, true));
+
+  let words = vec![10, 10];
```

Comment:
This is just a test so 🤷 .
In "real" code I'd reject this - allocating a `Vec` here is unnecessary.